lib/reducers/bwallet.js

```diff
@@ -6,6 +6,7 @@
 const initialState = {};
 
 export default function bwalletReducer(state = initialState, action) {
+  // eslint-disable-next-line
```

Comment:
what are we disabling? My preference for eslint disables is that we're explicit about what rule is being disabled.

Reply:
`payload` is unused, since this is still a skeleton

Reply:
gotcha. I would still use `// eslint-disable-next-line no-unused-vars` to make sure this doesn't catch anything else (and also clarify what was being ignored)

Reply:
Just pushed that up